lib/ansible/modules/cloud/docker/docker_network.py

```diff
@@ -289,26 +366,32 @@ def has_different_config(self, net):
 
     def create_network(self):
         if not self.existing_network:
+            params = dict()
```

Comment:
The reason why I did not include those two options is because they have a default. But changing is not a problem. Should I change it to
```
if self.parameters.driver is not None:
    params['driver'] = self.parameters.driver
if self.parameters.driver_options is not None:
   params['options'] = self.parameters.driver_options
```
or without conditionals?
```
params['driver'] = self.parameters.driver
params['options'] = self.parameters.driver_options
```

Reply:
The second, or simply even do
```
params = dict(
    driver=self.parameters.driver,
    options=self.parameters.driver_options,
)
```